### DISCUSSION TITLE
List of existing implemenations

### DISCUSSION THREAD

**Discussion by mj-will** [lscsoft/bilby#879] [HoM relative binning]
This is a follow on from https://github.com/bilby-dev/bilby/issues/796.

So far, it seems like there are few different implementations. In no particular order, here are the ones I have links to:

- Version by @narolaharsh and others: https://github.com/lemnis12/relativebilbying
- Version by Ish Gupta - missing link
- Version by @noahewolfe

It would be good to try and restart the discussion on this.

**Comment by corvus314** [comment]
Implementation by @narolaharsh is waveform specific. It was constructed for PhenomXPHM as it had to reimplement parts of the waveform generation outside of lalsimulation. Not sure if other implementations are more general.

**Reply by narolaharsh** [reply]
Right, the implementation inside relativebilbying is only for PhenomXPHM. I am a bit busy completing the analysis for the upcoming TGR paper but after that, I'd be happy to put together a merger request for this or work with others.

**Comment by adivijaykumar** [comment]
Tagging Ish (@img5156) on this issue as well.
